src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -302,8 +302,9 @@ SHAMapStoreImp::copyNode(
     SHAMapAbstractNode const& node)
 {
     // Copy a single record from node to dbRotating_
-    dbRotating_->fetchNodeObject(
-        node.getNodeHash().as_uint256(), node.getSeq());
+    // FIXME: node.owner() is the wrong thing to pass: fetchNodeObject expects
+    //        a ledger sequence, not a copy-on-write identifier.
+    dbRotating_->fetchNodeObject(node.getHash().as_uint256(), node.owner());
```

Comment:
After mentioning this bug in chat, it was fixed by @miguelportilla in #3643 so it will be resolved in a different PR, and this comment will go away during the merge/conflict resolution.